diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02b54b2..aa69329 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -100,7 +100,7 @@ if(PKG_INSTALL)
 endif()
 
 # The following will not be used for building a library. Just an application.
-if(BUILD_EXAMPLE)
+if(OFF)
     add_executable(development_and_deploy_main main.cpp)
     target_link_libraries(development_and_deploy_main
         PRIVATE 
